src/SFA.DAS.EmployerCommitmentsV2.Web/Views/DraftApprenticeship/SelectOption.cshtml

```diff
@@ -0,0 +1,91 @@
+﻿@using SFA.DAS.CommitmentsV2.Shared.Extensions
+@model SFA.DAS.EmployerCommitmentsV2.Web.Models.DraftApprenticeship.SelectOptionViewModel
+
+@{
+    ViewData["Title"] = "Select a standard option";
+}
+
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+        <partial name="_ValidationSummary" />
+
+        <h1 class="govuk-heading-xl">Select a standard option</h1>
+
+        @if (!string.IsNullOrEmpty(Model.StandardUrl))
+        {
+            <div class="govuk-inset-text">
+                <p class="govuk-body">For more information visit <a class="govuk-link" href="@Model.StandardUrl" target="_blank">the IfATE page for this standard (opens in a new tab)</a>.</p>
+            </div>
+        }
+
+        <dl class="govuk-summary-list">
+            <div class="govuk-summary-list__row">
+                <dt class="govuk-summary-list__key"> Apprenticeship training course </dt>
+                <dd class="govuk-summary-list__value"> @Model.StandardTitle </dd>
+            </div>
+            <div class="govuk-summary-list__row">
+                <dt class="govuk-summary-list__key"> Version </dt>
+                <dd class="govuk-summary-list__value"> @Model.Version </dd>
+            </div>
+        </dl>
+
+    </div>
+</div>
+
+<form id="selectOption" method="post">
+    <div class="govuk-form-group @Html.AddClassIfPropertyInError(x => x.CourseOption, "govuk-form-group--error")">
+        <input type="hidden" asp-for="DraftApprenticeshipId" />
```

Comment:
Do we need to re-post everything from the previous page? if the draft apprenticeship gets saved in the previous add, can we just pull through the id here to retrieve it on the server side post?